neqo-transport/src/crypto.rs

```diff
@@ -1088,6 +1089,7 @@ impl CryptoStates {
         }
     }
 
+    #[cfg(not(feature = "fuzzing"))]
     #[cfg(test)]
```

Comment:
Maybe we should be more consistent about using two `#[cfg(...)]` statements vs. `#[cfg(all(...))]`.